AmplifyTools/AmplifyXcode/AmplifyXcode.xcodeproj/project.pbxproj

```diff
@@ -1117,7 +1117,7 @@
 
 /* Begin PBXFileReference section */
 		"AEXML::AEXML::Product" /* AEXML.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = AEXML.framework; sourceTree = BUILT_PRODUCTS_DIR; };
-		"AmplifyXcode::AmplifyXcode::Product" /* AmplifyXcode */ = {isa = PBXFileReference; lastKnownFileType = text; path = AmplifyXcode; sourceTree = BUILT_PRODUCTS_DIR; };
+		"AmplifyXcode::AmplifyXcode::Product" /* amplify-xcode */ = {isa = PBXFileReference; explicitFileType = "compiled.mach-o.executable"; path = "amplify-xcode"; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
Updated product name to match what we currently have in [Package.swift](https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyTools/AmplifyXcode/Package.swift#L13)